networking_oneview/ml2/drivers/oneview/synchronization.py

```diff
@@ -47,6 +47,7 @@ def start(self):
 
         heartbeat = loopingcall.FixedIntervalLoopingCall(self.synchronize)
         heartbeat.start(interval=3600, initial_delay=0)
+        self.delete_outdated_flat_mapped_networks()
```

Comment:
this check had to be in the mech_oneview initialize, not here, right? @ricardoas

Reply:
Yeah... we should concentrate all these checks in the initialize and fail with a proper message when loading the driver!